org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/formatting2/regionaccess/internal/serializer/RegionAccessTestLanguageSyntacticSequencer.java

```diff
@@ -91,8 +91,8 @@ else if (match_Parenthesized_LeftParenthesisKeyword_0_p.equals(syntax))
 	 *     'recursion'*
 	 *
 	 * This ambiguous syntax occurs at:
-	 *     (rule start) '6' ('(' ('unassigned' (Datatype | ID))?)+ 'fragment' (ambiguity) 'child' mixed=Mixed
-	 *     (rule start) '6' ('(' ('unassigned' (Datatype | ID))?)+ 'fragment' (ambiguity) fragName=ID
+	 *     (rule start) '6' ('(' ('unassigned' (ID | Datatype))?)+ 'fragment' (ambiguity) 'child' mixed=Mixed
```

Comment:
Why has this order changed?

Reply:
Because it was not really stable before. see https://github.com/eclipse/xtext-core/commit/c4b4c8e4a9d7ca9ef4dc942982ce45b329749388